bangladesh.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Amazing Net Ninja</title>
+</head>
+<body>  
+    <h4>Why you are awesome</h4>
+    <ol>
+        <li>Your videos are concise and to the point</li>
+        <li>You explain very well</li>
+    </ol>
+
+    <h4>Why you are not awesome</h4>
+    <ol>
+        <li>You talk way too fast. Sometimes it's harder for the non-English to grasp</li>
```

Comment:
well, I can't agree with you. He talks fine...
btw, I'm from BD too....😁
@TarifChandan